k8s/core/limitrange.go

```diff
@@ -0,0 +1,91 @@
+package core
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/fields"
+)
+
+// LimitRangeOps is an interface to perform k8s LimitRange operations
+type LimitRangeOps interface {
+	// GetLimitRange gets the secrets object given its name and namespace
```

Comment:
nit: replace `secrets object` with `limitrange object` for comments.

Reply:
fixed.